Extensions/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageLocalization/PageLocalization.jsx

```diff
@@ -39,6 +41,27 @@ class PageLocalization extends Component {
         this.setState({ Pages });
     }
 
+    removeLocaleFromState(cultureCode){
+        const {Locales} = this.state;
+        let index = Locales.findIndex(l => l.CultureCode == cultureCode);
+        if(index > -1){
+            Locales.splice(index, 1);
+            this.setState({ Locales });
+        }
+    }
+
+    onDeletePage(page){
+        let callback = () => {
+            this.removeLocaleFromState(page.CultureCode);
+            if(page.TabId === utils.getCurrentPageId()){
+                let panelId = window.$('.socialpanel:visible').attr('id');
+                utils.getUtilities().panelViewData(panelId, {tab: [0]});
+                window.top.location.href = utils.getDefaultPageUrl();
```

Comment:
is `window.location` the default choice to navigate the browser between pages?

Reply:
Yes